### DISCUSSION TITLE
Welcome to gears Discussions!

### DISCUSSION THREAD

**Discussion by QuirkyCort** [QuirkyCort/gears#47] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by nadinev6** [comment]
Hi, the robot motor movements are stuck when I load a custom world with objects. A custom world without objects seem to work fine though?

Also, what files should be zipped for players in the GearsBot Arena, a custom robot and world?

**Reply by QuirkyCort** [reply]
Can you share the custom world? There may be something wrong with your file.

For the arena, you should do a "File -> Export zip package to your computer". This will export a zip file containing the blocks, python code, robot, and some metadata. You should not try to zip the files yourself.

**Reply by nadinev6** [reply]
Thanks for the reply 
[custom_world.txt](https://github.com/QuirkyCort/gears/files/6867776/custom_world.txt)

**Reply by QuirkyCort** [reply]
Your ground friction is set to zero. Changing it to 1 fixes the problem.

**Reply by nadinev6** [reply]
- ah thanks so much!

**Reply by nadinev6** [reply]
I'm having an issue with the objects. They don't seem to be movable or are they only meant to be obstacles?

**Reply by QuirkyCort** [reply]
Objects can be moveable or non-moveable. If you want it to be moveable, you'll need to either set the "physicsOptions" to "moveable", or set "physicsOptions" to "custom" and give the mass a non-zero value.

**Reply by nadinev6** [reply]
thanks, works now!
